templates/mdd/task3_qarecs/templates.yaml

```diff
@@ -0,0 +1,84 @@
+dataset: mdd
+subset: task3_qarecs
+templates:
+  1614890b-362c-4ee8-850d-841cf511d169: !Template
+    id: 1614890b-362c-4ee8-850d-841cf511d169
+    jinja: "Speaker 1: {{dialogue_turns.utterance[0]}}
```

Comment:
Can you add some sentences that describe the task? something like `This is a dialogue between Speaker 1 and Speaker 2. Complete Speaker 2's intervention to answer Speaker 1's question`.

Nit: is it always alternating between speaker 1 and speaker 2?

Reply:
Yes, I'll add this description.

And yes, it is always alternating between speaker 1 and speaker 2 and both get an equal number of utterances.